### ISSUE TITLE
REQUEST: Tough as Nails support for your Space Suit

### ISSUE THREAD

**Issue by mcdodge34** [zmaster587/AdvancedRocketry#794]
I discovered Tough as Nails while playing Forever Stranded modpack and I think it would probably fit well with some planets and some space stations of your mod, some planets could be very hot and force you to have protection in your suit to avoid overheating, same goes for extremely cold planets maybe, also a thirst module would perhaps be a good idea. Now the thing is to have the guys that develop Tough As Nails allow us to have whitelist and blacklist for dimensions so we could decide where the mod is active and where its not, that would make a great addition I think.

**Comment by Mementh** [comment]
if you know the dimensions.. then you could whitelist them in that mod? 
can you specify the temp though in a white list?

**Comment by mcdodge34** [comment]
Nope, its one of the main problem right now, it works only with the biomes, the temperature is different depending on the biomes and the mod author doesn't seem to want to add an option based on dimension sadly, which is why I was asking here, since its based on biomes, perhaps you could do something on your side, but I'm not 100% sure.
